README.adoc

```diff
@@ -260,16 +260,33 @@ Test out the build by executing CLI commands in a different terminal.
 
 NOTE: The CLI needs the API Server which was started on executing `make dev`  to be up and running. Please do not kill the process. Alternatively if you haven't run `make dev` you could just start the server by running `./bin/alm`.
 
-Create a work item.
+Create a work item type.
+----
+./bin/alm-cli create workitemtype --payload '{"extendedTypeName":null,"fields":{"system.owner":{"Type":{"Kind":"user"},"Required":true},"system.state":{"Type":{"Kind":"string"},"Required":false}},"name":"Epic"}' -H localhost:8080 --pp
+----
+Note: Work Item Type `Name` is unique. If one tries to create another work item type with same name, error will be trown.
+
+Retrieve the work item type.
+
+----
+$ ./bin/alm-cli show workitemtype --name "Epic" -H localhost:8080
+----
+
+List all available work item types.
+----
+$ ./bin/alm-cli list workitemtype -H localhost:8080 --pp
+----
+
+Create a work item. We need to use name of work item type in the `type` field below.
 
 ----
-$ ./bin/alm-cli create workitem --payload '{"type": "1" , "name": "some name", "fields": { "system.owner": "tmaeder", "system.state": "open" }}' -H localhost:8080
+$ ./bin/alm-cli create workitem --payload '{"type": "Epic" , "name": "some name", "fields": { "system.owner": "tmaeder", "system.state": "open" }}' -H localhost:8080
```

Comment:
@alexeykazakov @pranavgore09 The WorkItem definition match the WorkitemType created further up, so not wrong in that sense. e.g. system.state is just a string. 

@pranavgore09 If you add some text like: Based on the WorkItemType we created above, we can now create a WorkItem...